src/analysis/max_min_driving_force.jl

```diff
@@ -94,13 +94,15 @@ function max_min_driving_force(
     end
 
     # set proton log concentration to zero so that it won't impact any calculations (biothermodynamics assumption)
-    proton_idxs = Int.(indexin(proton_ids, metabolites(model)))
+    _proton_idxs = indexin(proton_ids, metabolites(model))
+    proton_idxs = isempty(_proton_idxs) ? Int[] : Int.(_proton_idxs)
```

Comment:
why is this needed? Int. should do this conversion right, is there a case that fails?

Reply:
this is in case the user doesn't have any protons or water in their model (as I did for a simple test model) :)

Reply:
but this types OK for me?
```julia
julia> Int.(indexin([], [1,2,3]))
Int64[]
```

Reply:
uh maybe what about this:
```julia
julia> Int.(filter(!isnothing, indexin([5,6], [1,2,3])))
Int64[]
```

Reply:
Nope, this was a user error, the function works fine as is. Reverting